PrioritizedSubPub.js

```diff
@@ -206,12 +206,17 @@
      * @desc private console output. Currently logs by default.
      * @todo: think of good way to toggle logging.
      */
-     function _debugLog() {
+    function _debugLog() {
         var args;
-        if (console && console.log) {
+        if (window.console && console.log) {
```

Comment:
I am going to have to try something like `var console = window.console || console;` and use the var instead.